src/main/java/teammates/ui/controller/InstructorStudentRecordsPageAction.java

```diff
@@ -90,7 +93,9 @@ public ActionResult execute() throws EntityDoesNotExistException {
         InstructorStudentRecordsPageData data =
                                         new InstructorStudentRecordsPageData(account, student, courseId,
                                                                              showCommentBox, studentProfile,
-                                                                             comments, sessionNames, instructor);
+                                                                             giverEmailToCommentsMap,
+                                                                             giverEmailToGiverNameMap,
+                                                                             sessionNames, instructor);
```

Comment:
Maybe rewrite it like so (so that it won't be skewed to the right)
```
InstructorStudentRecordsPageData data =
        new InstructorStudentRecordsPageData(
                account, student, courseId, showCommentBox, studentProfile,
                giverEmailToCommentsMap, giverEmailToGiverNameMap, sessionNames, instructor);
```